tests/client.test.ts

```
@@ -1,48 +1,70 @@
+  beforeEach(() => {
     process.env.SHORTCUT_API_KEY = 'fakeEnvKey'
   })
 
   afterAll(() => {
     jest.restoreAllMocks()
   })
 
-  it('creates new instance with shortcutApiKey', () => {
+  /** The client is private by design; tests reach in to bind a mock adapter to it. */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const adapterFor = (client: Client): AxiosMockAdapter => new AxiosMockAdapter((client as any).http)
```

ESLint

[error] Prefer a function declaration over an arrow function assigned to 'adapterFor'.

(yenz/no-named-arrow-functions)